libbeat/outputs/elasticsearch/api_integration_test.go

```diff
@@ -81,7 +81,7 @@ func TestIngest(t *testing.T) {
 		t.Skip("Skipping tests as pipeline not available in 2.x releases")
 	}
 
-	_, _, err := client.DeletePipeline(pipeline, nil)
+	client.DeletePipeline(pipeline, nil)
```

Comment:
Instead of ignoring the `err`, how about checking the error? I think the only acceptable status code would be 404 if the pipeline did not exist.